safe_transaction_service/history/services/collectibles_service.py

```diff
@@ -160,55 +164,73 @@ def __init__(self, ethereum_client: EthereumClient, redis: Redis):
             maxsize=4096, ttl=60 * 30
         )  # 2 hours of caching
 
+    def get_redis_metadata_timeout_key(self, uri: str):
+        return f"metadata_timeout:{uri}"
+
     @cachedmethod(cache=operator.attrgetter("cache_uri_metadata"))
     @cache_memoize(
```

Comment:
I would like remove this decorator and do it with django_cache, what do you think @Uxio0 ?

Reply:
The second exception shouldn't be necessary if we manage the cache without the decorator.